components/engine_panic/src/raft_engine.rs

```diff
@@ -4,6 +4,7 @@ use crate::engine::PanicEngine;
 use crate::write_batch::PanicWriteBatch;
 use engine_traits::{Error, RaftEngine, RaftEngineReadOnly, RaftLogBatch, Result};
 use kvproto::raft_serverpb::RaftLocalState;
+
```

Comment:
🤔

Reply:
This was introduced by the `1bd899762d`, which is a merge master commit.